include/functions.h

```diff
@@ -44,7 +44,8 @@ void runAnimalCleanup(GObj* obj);
 GObj* addAnimalAtGeo(GObj* obj, u16 AnimalID, animalDef* def);
 roomGFX* setNodePosToNegRoom(GObj*);
 void animalPathLoop(GObj* obj, f32 start, f32 end, f32 dt, f32 yawStep, u32 flags);
-void spawnAnimalUsingDeltaHeight(s32 gObjID, u16 id, roomGFX* roomA, roomGFX* roomB, objectSpawn* spawn, animalInitData* initData);
+void spawnAnimalUsingDeltaHeight(s32 gObjID, u16 id, roomGFX* roomA, roomGFX* roomB, objectSpawn* spawn,
```

Comment:
Would rather see this like
```
void spawnAnimalUsingDeltaHeight(
    s32 gObjID,
    u16 id,
    roomGFX* roomA,
    roomGFX* roomB,
    objectSpawn* spawn,
    animalInitData* initData
);

Reply:
I agree for long arg lists but not in general (not sure if this is configurable based on line length)